bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/PersistentInbox.java

```diff
@@ -126,6 +126,8 @@ private boolean isResultExpired(DiscoveryResult result, long now) {
 
     private Storage<DiscoveryResult> discoveryResultStorage;
 
+    private Map<DiscoveryResult, String> resultDiscovererMap = new HashMap<>();
```

Comment:
I don't care much, but wouldn't it be simpler to put the Class itself `Class<?>` to the list instead of using the FQN of the class?